app/app.go

```diff
@@ -494,6 +494,13 @@ func NewAxelarApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 		permission.NewAppModule(permissionK),
 	)
 
+	app.mm.SetOrderMigrations(
```

Comment:
set begin/end blocker needs include all modules, not sure it's the same here

Reply:
it is the same. Fixed